app/models/map-features/lot.js

```diff
@@ -426,16 +422,16 @@ export default class LotFragment extends MF.Fragment {
     @computed('cd')
     get cdName() {
       const borocd = this.cd;
-      const boro = `${borocd}`.substring(0, 1);
+      const borocode = this.borocode; // eslint-disable-line prefer-destructuring
       const cd = parseInt(`${borocd}`.substring(1, 3), 10).toString();
-      return `${boroLookup[boro]} Community District ${cd}`;
+      return `${boroLookup[borocode]} Community District ${cd}`;
```

Comment:
I thought we changed this to use the first digit of borocd for just this use case because of cases like Marble Hill being in a Bronx CD even though it's in Manhattan? And, side note, let's try to avoid turning of eslint rules if we can. Was this one giving you particular trouble for some reason?

Reply:
I recall we tried this to get the test to pass. After the test failure, I changed it back to use the `borocode` column. The tests passed with this  but according to CD profiles, Marble Hill, Manhattan is part of the Bronx CD8 🥴   
I changed the const to `cdborocode` to more accurately describe what it is, `borocode` didn't seem accurate to me since MH is in Manhattan.